exporter.php

```diff
@@ -0,0 +1,382 @@
+		if ( $t->parent == 0 || isset( $terms[ $t->parent ] ) ) {
+			$terms[ $t->term_id ] = $t;
+		} else {
+			$custom_terms[] = $t;
+		}
+	}
+
+	unset( $categories, $custom_taxonomies, $custom_terms );
+}
+
+function p4_px_single_post_cdata( $str ) {
+	if ( seems_utf8( $str ) == false ) {
+		$str = utf8_encode( $str );
+	}
+	$str = '<![CDATA[' . str_replace( ']]>', ']]]]><![CDATA[>', $str ) . ']]>';
+
+	return $str;
+}
+
+function p4_px_single_post_site_url() {
+	if ( is_multisite() ) {
+		return network_home_url();
+	} else {
+		return get_bloginfo_rss( 'url' );
+	}
+}
+
+function p4_px_single_post_cat_name( $category ) {
+	if ( empty( $category->name ) ) {
+		return;
+	}
+
+	echo '<wp:cat_name>' . p4_px_single_post_cdata( $category->name ) . '</wp:cat_name>';
+}
+
+function p4_px_single_post_category_description( $category ) {
+	if ( empty( $category->description ) ) {
+		return;
+	}
+
+	echo '<wp:category_description>' . p4_px_single_post_cdata( $category->description ) . '</wp:category_description>';
+}
+
+function p4_px_single_post_tag_name( $tag ) {
+	if ( empty( $tag->name ) ) {
+		return;
+	}
+
+	echo '<wp:tag_name>' . p4_px_single_post_cdata( $tag->name ) . '</wp:tag_name>';
+}
+
+function p4_px_single_post_tag_description( $tag ) {
+	if ( empty( $tag->description ) ) {
+		return;
+	}
+
+	echo '<wp:tag_description>' . p4_px_single_post_cdata( $tag->description ) . '</wp:tag_description>';
+}
+
+function p4_px_single_post_term_name( $term ) {
+	if ( empty( $term->name ) ) {
+		return;
+	}
+
+	echo '<wp:term_name>' . p4_px_single_post_cdata( $term->name ) . '</wp:term_name>';
+}
+
+function p4_px_single_post_term_description( $term ) {
+	if ( empty( $term->description ) ) {
+		return;
+	}
+
+	echo '<wp:term_description>' . p4_px_single_post_cdata( $term->description ) . '</wp:term_description>';
+}
+
+function p4_px_single_post_authors_list( $post_ids ) {
+	global $wpdb;
+
+	$post_ids = implode( ',', $post_ids );
+
+	$authors = array();
+	$results = $wpdb->get_results( "SELECT DISTINCT post_author FROM $wpdb->posts WHERE ID IN ( $post_ids ) AND post_status != 'auto-draft'" );
+	foreach ( (array) $results as $result ) {
+		$authors[] = get_userdata( $result->post_author );
+	}
+
+	$authors = array_filter( $authors );
+
+	foreach ( $authors as $author ) {
+		echo "\t<wp:author>";
+		echo '<wp:author_id>' . $author->ID . '</wp:author_id>';
+		echo '<wp:author_login>' . $author->user_login . '</wp:author_login>';
+		echo '<wp:author_email>' . $author->user_email . '</wp:author_email>';
+		echo '<wp:author_display_name>' . p4_px_single_post_cdata( $author->display_name ) . '</wp:author_display_name>';
+		echo '<wp:author_first_name>' . p4_px_single_post_cdata( $author->user_firstname ) . '</wp:author_first_name>';
+		echo '<wp:author_last_name>' . p4_px_single_post_cdata( $author->user_lastname ) . '</wp:author_last_name>';
+		echo "</wp:author>\n";
+	}
+}
+
+function p4_px_single_post_nav_menu_terms() {
```

Comment:
Do we need support for nav menu terms in this export file?

Reply:
Don't believe so. Removed.